src/sanity/schemaTypes/statisticsCard.ts

```diff
@@ -0,0 +1,59 @@
+import { defineType, defineField } from "sanity";
+import { ClipboardImageIcon } from "@sanity/icons";
+
+export const StatisticsCard = defineType({
+  name: "statisticsCard",
+  title: "Statistics Card",
+  icon: ClipboardImageIcon,
+  type: "document",
+  fields: [
+    defineField({
+      name: "image",
+      title: "Image",
+      type: "image",
+      fields: [
+        {
+          name: "alt",
+          type: "string",
+          title: "Alternative text",
+          description: "Important for SEO and accessibility.",
+          validation: (rule) => {
+            return rule.custom((alt, context) => {
+              if ((context.document?.picture as any)?.asset?._ref && !alt) {
+                return "Required";
+              }
+              return true;
+            });
+          },
+        },
+      ],
+      options: {
+        aiAssist: {
+          imageDescriptionField: "alt",
+        },
+      },
+      validation: (rule) => rule.required(),
+    }),
+    defineField({
```

Comment:
Was it necessary to make it a **block** type? Or did you do it to make it more flexible in case they want to change the style/weight/... in the future?